lib/scrabble.rb

```diff
@@ -1,18 +1,23 @@
 class Scrabble
+  def initialize
+    @game_reader = GameReader.new
+  end
```

Comment:
This feels like an unnecessary complication. Looking at the interaction pattern, these methods are implemented directly on the Scrabble class. No need for a GameReader class. More straightforward than that: this doesn't follow the spec.